app/services/notify_service.rb

```diff
@@ -65,7 +65,12 @@ def create_notification
   end
 
   def send_push_notifications
-    WebPushNotificationWorker.perform_async(@recipient.id, @notification.id)
+    sessions_with_subscriptions = @recipient.user.session_activations.where.not(web_push_subscription: nil)
+
+    Sidekiq::Client.push_bulk(
+      'class' => WebPushNotificationWorker,
+      'args' => sessions_with_subscriptions.map { |session_activation| [session_activation.id, @notification.id] }
```

Comment:
That implementation was fine. It uses `select(:id)`. And it gives a block for `push_bulk`, which would be better than calling `map`.